index.js

```diff
@@ -580,7 +580,7 @@ class Grid extends react.Component {
  * @param {number} page The query page number
  * @returns Array of search results (filtered through the blacklist)
  */
-async function getRepos(page = 1) {
+async function getExtensionRepos(page = 1) {
```

Comment:
Let's call this guy `fetchExtensionRepos` to keep it the same as `fetchThemeRepos`